systemd-swap

```diff
@@ -70,7 +70,16 @@ get_fs_type(){
   elif [ -d $(dirname "$1") ]; then path=$(dirname "$1");
   else ERRO "swapfc_path is invalid"
   fi
-  df "$path" --output=fstype | tail -n 1
+
+  fstype=$(df "$path" --output=fstype | tail -n 1)
+
+  if [ "$fstype" == "-" ] && btrfs subvolume show "$path" &>/dev/null; then
+    fstype="btrfs"
+  else
+    ERRO '$swapfc_path is located on an unknown filesystem'
+  fi
```

Comment:
So if `$fstype` is ext4 this results in an error?

*Note: we should avoid whilelists, so an `elif fsype = ext4/xfs/etcetera` is not a very elegant solution.